docs/sources/developers/http_api/dashboard_public.md

```diff
@@ -0,0 +1,315 @@
+---
+aliases:
+  - ../../http_api/dashboard_public/
+canonical: /docs/grafana/latest/developers/http_api/dashboard_public/
+description: Grafana Public Dashboard HTTP API
+keywords:
+  - grafana
+  - http
+  - documentation
+  - api
+  - dashboard
+labels:
+  products:
+    - enterprise
+    - oss
+title: Public Dashboard HTTP API
+---
+
+# Public Dashboard API
+
+> If you are running Grafana Enterprise, for some endpoints you'll need to have specific permissions. Refer to [Role-based access control permissions]({{< relref "/docs/grafana/latest/administration/roles-and-permissions/access-control/custom-role-actions-scopes" >}}) for more information.
+
+## Create a public dashboard
+
+`POST /api/dashboards/uid/:uid/public-dashboards/`
+
+Creates a new public dashboard
+
+**Required permissions**
+
+See note in the [introduction]({{< ref "#public-dashboard-api" >}}) for an explanation.
+
+| Action                    | Scope                            |
+| ------------------------- | -------------------------------- |
+| `dashboards.public:write` | `dashboards:uid:<dashboard UID>` |
+
+**Example Request for new public dashboard**:
+
+```http
+POST /api/dashboards/uid/xCpsVuc4z/public-dashboards/ HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+
+{
+    "uid": "cd56d9fd-f3d4-486d-afba-a21760e2acbe",
+    "accessToken": "5c948bf96e6a4b13bd91975f9a2028b7",
+    "timeSelectionEnabled": false,
+    "isEnabled": true,
+    "annotationsEnabled": false,
+    "share": "public"
+}
+```
+
+JSON Body schema:
+
+- **uid** – Optional. Unique identifier when creating a public dashboard. If it is null it will generate a new uid.
+- **accessToken** – Optional. Unique access token. If it is null it will generate a new access token.
+- **timeSelectionEnabled** – Optional. Enable the time picker in the public dashboard. Default value is false.
+- **isEnabled** – Optional. Enable the public dashboard. Default value is false.
+- **annotationsEnabled** – Optional. Show annotations. Default value is false.
+- **share** – Optional. Share mode. Default value is public.
```

Comment:
```suggestion
- **uid** – Optional. Unique identifier when creating a public dashboard. If it's null, it will generate a new uid.
- **accessToken** – Optional. Unique access token. If it's null, it will generate a new access token.
- **timeSelectionEnabled** – Optional. Set to `true` to enable the time picker in the public dashboard. The default value is `false`.
- **isEnabled** – Optional. Set to `true` to enable the public dashboard. The default value is `false`.
- **annotationsEnabled** – Optional. Set to `true` to show annotations. The default value is `false`.
- **share** – Optional. Set the share mode. The default value is `public`.
```